frontend/components/InlineFeedbackPopover/Comment.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react'
+import {
+  useUpdateCommentMutation,
+  useDeleteCommentMutation,
+  CommentFragmentFragment as CommentType,
+} from '../../generated/graphql'
+import Button, { ButtonSize, ButtonVariant } from '../../elements/Button'
+import BlankAvatarIcon from '../Icons/BlankAvatarIcon'
+import theme from '../../theme'
+
+type CommentProps = {
+  comment: CommentType
+  canEdit: boolean
+  onUpdateComment: any
+}
+
+const Comment: React.FC<CommentProps> = ({ comment, canEdit, onUpdateComment }) => {
+  const [isEditMode, setIsEditMode] = React.useState<boolean>(false)
+  const [updatingCommentBody, setUpdatingCommentBody] = useState<string>(comment.body)
+
+  const [updateComment, { loading }] = useUpdateCommentMutation({
+    onCompleted: () => {
+      onUpdateComment()
+      setUpdatingCommentBody('')
+    },
+  })
+
+  const updateExistingComment = () => {
+    updateComment({
+      variables: {
+        commentId: comment.id,
+        body: updatingCommentBody,
+      },
+    })
+    setIsEditMode(false)
+  }
+
+  const [deleteComment, { loading: deleteLoading }] = useDeleteCommentMutation({
+    onCompleted: () => {
+      // just refetches the post as in updateComment
+      onUpdateComment()
+    },
+  })
+
+  const deleteExistingComment = () => {
+    deleteComment({
+      variables: {
+        commentId: comment.id,
+      },
+    })
+  }
+
+  return (
+    <div className="comment">
+      <div className="author-block">
+        {comment.author.profileImage ? (
+          <img className="profile-image" src={comment.author.profileImage} alt="" />
+        ) : (
+          <BlankAvatarIcon size={20} />
+        )}
+        <span className="author-identifier">
+          {comment.author.name
+            ? `${comment.author.name} (@${comment.author.handle})`
+            : `@${comment.author.handle}`}
+        </span>
+      </div>
+      <div className="body-block">
+        {isEditMode ? (
+          <textarea
+            value={updatingCommentBody}
+            onChange={(e) => setUpdatingCommentBody(e.target.value)}
+          />
+        ) : (
+          <p>{comment.body}</p>
+        )}
+        {canEdit && !isEditMode && (
+          <div className="edit-block">
+            <Button size={ButtonSize.Small} onClick={() => setIsEditMode(true)}>
+              edit
```

Comment:
Maybe `Edit` (and `Delete` / `Save` on lines 87 and 93) to match the capitalization of `Submit` and the other buttons?

Reply:
Sounds good to meeee, done!